docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+version: "3.7"
+services:
+  app:
+    build:
+      args:
+        user: daskom
+        uid: 1000
+      context: ./
+      dockerfile: Dockerfile
+    image: daskomlab
+    container_name: daskomlab-app
+    restart: unless-stopped
+    working_dir: /var/www/
+    ports:
+      - 9000:9000
+      - 6001:6001
+    volumes:
+      - ./:/var/www
+    networks:
+      - laravel
+
+  db:
+    image: mysql:5.7.22
+    container_name: daskomlab-db
+    restart: unless-stopped
+    tty: true
+    ports:
+      - 33061:3306
+    environment:
+      MYSQL_DATABASE: ${DB_DATABASE}
+      MYSQL_ROOT_PASSWORD: ${DB_PASSWORD}
+      MYSQL_PASSWORD: ${DB_PASSWORD}
+      MYSQL_USER: ${DB_USERNAME}
```

Comment:
Make sure variables and setup is for **deployment** environment instead of local

Reply:
okay, thankyou for reminder ^-^

Reply:
Resolve when this has been checked okay!!